packages/frontend/core/src/components/tags/tag.css.ts

```diff
@@ -151,6 +152,7 @@ export const tagLabel = style({
   textOverflow: 'ellipsis',
   whiteSpace: 'nowrap',
   userSelect: 'none',
+  transform: 'translateY(-0.75px)',
```

Comment:
too specific? what happened to the tags' vertical alignment?

Reply:
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/T2klNLEk0wxLh4NRDzhk/c00527b6-04d4-4fc5-a254-c361092754ba.png)

The text of the tag label's baseline should align with the bottom of the "dot" on the left